pkgs/development/libraries/libvirt/default.nix

```diff
@@ -66,45 +66,46 @@ in stdenv.mkDerivation rec {
     substituteInPlace src/lxc/lxc_conf.c \
       --replace 'lxc_path,' '"/run/libvirt/nix-emulators/libvirt_lxc",'
     patchShebangs . # fixes /usr/bin/python references
-    mkdir -p build && cd build
   '';
 
-  configureScript = "../configure";
-
   dontAddDisableDepTrack = true;
 
-  configureFlags = [
-    "--with-runstatedir=/run" # TODO: remove when autoconf 2.70 is released
-    "--localstatedir=/var"
-    "--sysconfdir=/var/lib"
-    "--with-libpcap"
-    "--with-qemu"
-    "--with-vmware"
-    "--with-vbox"
-    "--with-test"
-    "--with-esx"
-    "--with-remote"
-    "--with-polkit"
+  mesonFlags = let
+    opt = option: enable: "-D${option}=${if enable then "enabled" else "disabled"}";
+  in [
+    "-Dauto_features=disabled"
+    "-Drunstatedir=/run"
+    "-Dlocalstatedir=/var"
+    "-Dsysconfdir=/var/lib"
```

Comment:
This is weird, `--sysconfdir` should typically be `/etc`. Also note that you might need a patch like [this one](https://github.com/NixOS/nixpkgs/blob/fd950b9fc7bc97f38b5583bd08e3840834ea6712/pkgs/os-specific/linux/firmware/fwupd/add-option-for-installation-sysconfdir.patch) since ninja does not allow configuring variables at installation time.